raster/src/main/scala/geotrellis/raster/io/geotiff/LazySegmentBytes.scala

```diff
@@ -29,6 +29,11 @@ class LazySegmentBytes(
 ) extends SegmentBytes with LazyLogging {
   import LazySegmentBytes.Segment
 
+  // TODO: make it a parameter in GeoTrellis 2.0
+  // max distance between two segments in a group
+  // used in a chunkSegments function
+  private val minChunkSize: Int = 1024
```

Comment:
Is there an issue for this, or some way to track this TODO? Is it possible to use deprecation or overloads to solve this?

Reply:
I'll try some variants, mb it's possible smth, I agree that we can forget about it